### DISCUSSION TITLE
forward and inverse search in .qmd?

### DISCUSSION THREAD

**Discussion by sje30** [quarto-dev/quarto-cli#2163] [Feature Requests]
One great feature of the .tex based workflow from knitr is the ability to have forward and inverse search:

https://support.rstudio.com/hc/en-us/articles/200552066-PDF-Preview-and-SyncTeX

That page notes that the feature is not available for rmarkdown or quarto docs.  But just out of interest, how difficult would it be to get forward and inverse searching between .qmd and .pdf for example?  Too tricky to maintain the concordance files?

**Comment by cderv** [comment]
Hi, 

This seems like IDE related feature. 

Are you looking for this for RStudio IDE ? or rather VSCODE ? 

Opening a feature request in one of those places would be more efficient to track the idea I think

**Reply by sje30** [reply]
thanks -- would primarily be using this from Emacs :-) so much older tech...

my thinking was that the concordance file between .qmd and e.g. the PDF would be independent of the IDE/GUI -- in which case, it would be a core feature.   but happy for this to be closed if there is not much interest in the topic at the moment.

**Reply by FelixBenning** [reply]
https://stackoverflow.com/q/70978634/6662425 This seems to be a feature that is non-existent in general for html. Since synctex is also an extra file, it may be that it is first necessary to store the correspondence somewhere first (e.g. between html tag ids and lines of code).

**Comment by cderv** [comment]
Related issue opened at https://github.com/quarto-dev/quarto/issues/136
